doc/source/examples/04_applied/02_naca_fluent.mystnb

```diff
@@ -0,0 +1,150 @@
+
+# Retrieve the airfoil body
+airfoil = design.bodies[0]
+
+# Display the airfoil
+design.plot()
+```
+
+## Prepare the geometry for the simulation
+
+The current design is only composed of the airfoil. To prepare the geometry for the simulation, it
+is necessary that you define the domain around the airfoil. The following code creates a rectangular
+fluid domain around the airfoil.
+
+The airfoil has the following dimensions:
+
+* Chord length: 1 (X-axis)
+* Thickness: depends on NACA value (Y-axis)
+
+The fluid domain will be a large box with the following dimensions:
+
+* Length (X-axis) - 10 times the chord length
+* Width (Z-axis) - 5 times the chord length
+* Height (Y-axis) - 4 times the chord length
+
+The airfoil will be placed at the center of the fluid domain.
+
+```{code-cell} ipython3
+from ansys.geometry.core.math import Point2D, Plane, Point3D
+from ansys.geometry.core.sketch import Sketch
+
+BOX_LENGTH = 10  # X-Axis
+BOX_WIDTH = 5  # Z-Axis
+BOX_HEIGHT = 4  # Y-Axis
+
+# Create the sketch
+fluid_sketch = Sketch(
+    plane=Plane(origin=Point3D([0, 0, 0.5 - (BOX_WIDTH / 2)]))
+)
+fluid_sketch.box(
+    center=Point2D([0.5, 0]),
+    height=BOX_HEIGHT,
+    width=BOX_LENGTH,
+)
+
+# Extrude the fluid domain
+fluid = design.extrude_sketch("Fluid", fluid_sketch, BOX_WIDTH)
+```
+
+## Creating named selections
+
+Named selections are used to define boundary conditions in Fluent. The following code creates named selections for the
+inlet, outlet, and walls of the fluid domain. The airfoil is also assigned a named selection.
+
+The airfoil is aligned with the X-axis. The inlet is located at the left side of the airfoil, the outlet is located at the
+right side of the airfoil, and the walls are located at the top and bottom of the airfoil. The inlet face has therefore
+a negative X-axis normal vector, and the outlet face has a positive X-axis normal vector. The rest of the faces, therefore,
+constitute the walls.
+
+```{code-cell} ipython3
+# Create named selections in the fluid domain - inlet, outlet, and surrounding faces
+# Add also the airfoil as a named selection
+fluid_faces = fluid.faces
+surrounding_faces = []
+inlet_faces = []
+outlet_faces = []
+for face in fluid_faces:
+    if face.normal().x == 1:
+        outlet_faces.append(face)
+    elif face.normal().x == -1:
+        inlet_faces.append(face)
+    else:
+        surrounding_faces.append(face)
+
+design.create_named_selection("Outlet Fluid", faces=outlet_faces)
+design.create_named_selection("Inlet Fluid", faces=inlet_faces)
+design.create_named_selection("Surrounding Faces", faces=surrounding_faces)
+design.create_named_selection("Airfoil Faces", faces=airfoil.faces)
+```
+
+## Display the geometry
+
+The geometry is now ready for the simulation. The following code displays the geometry in the notebook.
+This example uses the ``PlotterHelper`` class to display the geometry with different opacity levels for
+the airfoil and fluid domain, as well as different colors.
```

Comment:
```suggestion
This example uses the ``PlotterHelper`` class to display the geometry for
the airfoil and fluid domain in different colors with a specified opacity level.
```
The order in this sentence and the next should be the same. (Color first, followed by opacity).